content/learn/book/welcome/app/_index.md

```diff
@@ -0,0 +1,82 @@
++++
+title = "Creating Bevy apps"
+weight = 2
+template = "book-section.html"
+page_template = "book-section.html"
++++
+
+Bevy programs store and execute all of their game logic and data with a single `App` data structure.
+Let's make a trivial Hello World app.
+
+```rust
+use bevy::prelude::*;
+
+fn main(){
+  App::new()
+  .add_system(hello.system())
+  .run();
+}
+
+fn hello(){
+    println!("Hello, Bevy!")
+}
+```
+
+The process is straightforward: we first create a new [`App`](https://docs.rs/bevy/latest/bevy/app/struct.App.html).
```

Comment:
"Bevy programs store and execute all of their game logic and data with a single `App` data structure."

The quoted text should go after this line, imho.